plugins/catalog/src/js/repositories/components/RepositoriesTable.js

```diff
@@ -223,21 +158,10 @@ class RepositoriesTable extends mixin(StoreMixin) {
           data={props.repositories.getItems().slice()}
           sortBy={{ prop: "priority", order: "asc" }}
         />
-        <Confirm
-          closeByBackdropClick={true}
-          disabled={state.pendingRequest}
-          header={heading}
-          open={!!state.repositoryToRemove}
+        <RepositoriesDelete
```

Comment:
That should be prop types, we could have written:
```
RepositoriesList.propTypes = {
 //...
}
```

As we did for `RepositoriesList.routeConfig`. I just overlooked on purpose because I was going to vacation and also because I plan to add typescript to it as I move forward to do the graphql implementation on how to fetch datra.